src/bp-forums/classes/class-bp-forums-component.php

```diff
@@ -424,5 +426,134 @@ public function bb_group_forums_set_title_tag( $title ) {
 
 			return $title;
 		}
+
+		/**
+		 * Admin bar menu for forum and topic.
+		 *
+		 * @since BuddyBoss [BBVERSION]
+		 */
+		public function bb_forums_admin_bar_menu() {
+			global $wp_admin_bar;
+			if ( bp_is_single_item() && bp_is_group() && bp_current_action() === 'forum' && ! bp_is_group_forum_topic() ) {
+				$query = new WP_Query(
+					array(
+						'name'      => get_query_var( 'name' ),
+						'post_type' => bbp_get_forum_post_type(),
+						'orderby'   => 'ID',
+						'order'     => 'ASC',
+					)
+				);
+				$forum = isset( $query->post ) ? $query->post : '';
+				if ( empty( $forum ) ) {
+					return;
+				}
+				$forum_id = isset( $forum->ID ) ? $forum->ID : '';
+			} else {
+				if ( bbp_is_single_forum() ) {
+					$forum_id = bbp_get_forum_id();
+				}
+			}
+			if ( bp_is_single_item() && bp_is_group() && bp_current_action() === 'forum' && bp_is_group_forum_topic() ) {
+				$query = new WP_Query(
+					array(
+						'name'      => bp_action_variable( 1 ),
+						'post_type' => bbp_get_topic_post_type(),
+						'orderby'   => 'ID',
+						'order'     => 'ASC',
+					)
+				);
+				$topic = isset( $query->post ) ? $query->post : '';
+				if ( empty( $topic ) ) {
+					return;
+				}
+				$topic_id = isset( $topic->ID ) ? $topic->ID : '';
+			} else {
+				if ( bbp_is_single_topic() ) {
+					$topic_id = bbp_get_topic_id();
+				}
+			}
+			if ( ! empty( $forum_id ) ) {
+				$wp_admin_bar->add_menu(
+					array(
+						'parent' => '',
+						'id'     => 'edit-forumn',
```

Comment:
Is this a typo or intentionally wrong written?
```suggestion
						'id'     => 'edit-forum',
```

Reply:
@chetansatasiya  I have fixed it.